providers/k8s/resources/ingress.go

```diff
@@ -60,16 +56,33 @@ func (k *mqlK8s) ingresses() ([]interface{}, error) {
 			"created":         llx.TimeData(ts.Time),
 			"manifest":        llx.DictData(manifest),
 			"rules":           llx.ArrayData(rules, types.Resource("k8s.ingressrule")),
-			"tls":             llx.ArrayData(tls, types.Resource("k8s.ingresstls")),
 		})
 		if err != nil {
 			return nil, err
 		}
 		r.(*mqlK8sIngress).obj = ingress
+		r.(*mqlK8sIngress).objId = objId
 		return r, nil
 	})
 }
 
+func (k *mqlK8sIngress) tls() ([]interface{}, error) {
+	o, err := CreateResource(k.MqlRuntime, "k8s", map[string]*llx.RawData{})
+	if err != nil {
+		return nil, err
+	}
+	k8s := o.(*mqlK8s)
+
+	ingress := k.obj
+	objId := k.objId
```

Comment:
nit: since these variables are only used once, you could just apply them

Reply:
Good point. Thanks.